content/MBC5.md

```diff
@@ -16,7 +16,7 @@ now. Also, bank 0 is actually bank 0.
 
 ### A000-BFFF - RAM Bank 00-0F, if any (Read/Write)
 
-Same as for MBC1, except RAM sizes are 64kbit, 256kbit and 1mbit.
+Same as for MBC1, except that RAM sizes are 8KiB, 32KiB and 128KiB.
```

Comment:
Spaces between the numbers and units, please

Reply:
Fixed